blueprint/index.md

```diff
@@ -84,9 +84,29 @@ The integration provides the following examples of Architect flows:
 
 ### HRIS-Get-Agents flow
 
-This HRIS provides a list of all agent records, including their corresponding IDs/keys and email addresses. Genesys Cloud workforce management requires this data to retrieve time-off balances and insert time-off requests for an agent.
+The primary purpose of this workflow is to retrieve the list of agents configured in external HRIS system and that information is used in Genesys's HRIS agent synchronization process that is scheduled to run once every 24 hours.
+
+This workflow is expected to return a list of emailIds and associated externalIds of the agents from external HRIS system that is to be synchronized with Genesys. 
```

Comment:
The number of records in both lists should match. The list order is important as the system will use the order to match elements in list of ids with the list of emails.

Reply:
will do